packages/data/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 -   Fix parsing bad JSON string data in useUserPreferences hook. #6819
+-   Removed allowed keys list for adding woocommerce_meta data. #6889
```

Comment:
Can you add this line to `readme.txt` under the `== Unreleased ==` header? That will stop the linter from failing.

Like:
`- Dev: Removed allowed keys list for adding woocommerce_meta data. #6889`